static/app/utils/useKeyPress.tsx

```diff
@@ -1,33 +1,39 @@
-import {useEffect, useState} from 'react';
+import {RefObject, useEffect, useState} from 'react';
 
 /**
  * Hook to detect when a specific key is being pressed
  */
-function useKeyPress(targetKey: React.KeyboardEvent['key']) {
+const useKeyPress = (targetKey: string, ref?: RefObject<HTMLInputElement>) => {
   const [keyPressed, setKeyPressed] = useState(false);
 
-  useEffect(() => {
-    function downHandler({key}: KeyboardEvent) {
-      if (key === targetKey) {
-        setKeyPressed(true);
-      }
+  const downHandler = ({key}: KeyboardEvent) => {
+    if (key === targetKey) {
+      setKeyPressed(true);
     }
+  };
 
-    function upHandler({key}: KeyboardEvent) {
-      if (key === targetKey) {
-        setKeyPressed(false);
-      }
+  const upHandler = ({key}: KeyboardEvent) => {
+    if (key === targetKey) {
+      setKeyPressed(false);
     }
-    window.addEventListener('keydown', downHandler);
-    window.addEventListener('keyup', upHandler);
+  };
+
+  useEffect(() => {
+    const current = ref?.current || window;
```

Comment:
```suggestion
    const current = ref?.current ?? window;
```

Probably makes more sense since it will be undefined